<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  
  
  
  
    
    
    
  
  

  

  
  
  
    
  
  <meta name="description" content="Chief Researcher Officer">

  
  <link rel="alternate" hreflang="en-us" href="/">

  


  
  
  
  <meta name="theme-color" content="rgb(0, 136, 204)">
  

  
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css" integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/github.min.css" crossorigin="anonymous" title="hl-light" disabled>
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark">
        
      
    

    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/css/academic.css">

  





<script async src="https://www.googletagmanager.com/gtag/js?id=UA-57219570-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];

  function gtag() {
      dataLayer.push(arguments);
  }

  function trackOutboundLink(url, target) {
    gtag('event', 'click', {
         'event_category': 'outbound',
         'event_label': url,
         'transport_type': 'beacon',
         'event_callback': function () {
           if (target !== '_blank') {
             document.location = url;
           }
         }
    });
    console.debug("Outbound link clicked: " + url);
  }

  function onClickCallback(event) {
    if ((event.target.tagName !== 'A') || (event.target.host === window.location.host)) {
      return;
    }
    trackOutboundLink(event.target, event.target.getAttribute('target'));  
  }

  gtag('js', new Date());
  gtag('config', 'UA-57219570-1', {});

  
  document.addEventListener('click', onClickCallback, false);
</script>


  


  
  
  <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Neil Dalchau">
  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/images/icon_huad7c223e4d485e0afbd0b521af84bf55_11903_32x32_fill_lanczos_center_3.png">
  <link rel="apple-touch-icon" type="image/png" href="/images/icon_huad7c223e4d485e0afbd0b521af84bf55_11903_192x192_fill_lanczos_center_3.png">

  <link rel="canonical" href="/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="Neil Dalchau">
  <meta property="og:url" content="/">
  <meta property="og:title" content="Neil Dalchau">
  <meta property="og:description" content="Chief Researcher Officer"><meta property="og:image" content="/images/icon_huad7c223e4d485e0afbd0b521af84bf55_11903_512x512_fill_lanczos_center_3.png">
  <meta property="twitter:image" content="/images/icon_huad7c223e4d485e0afbd0b521af84bf55_11903_512x512_fill_lanczos_center_3.png"><meta property="og:locale" content="en-us">
  
    <meta property="og:updated_time" content="2022-10-31T00:00:00&#43;00:00">
  

  

<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "WebSite",
  "potentialAction": {
    "@type": "SearchAction",
    "target": "/?q={search_term_string}",
    "query-input": "required name=search_term_string"
  },
  "url": "/"
}
</script>


  


  


  





  <title>Neil Dalchau</title>

</head>
<body id="top" data-spy="scroll" data-offset="70" data-target="#navbar-main" class="dark">

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  









<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/">Neil Dalchau</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/">Neil Dalchau</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about" data-target="#about"><span>Home</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#industryexperience" data-target="#industryexperience"><span>Experience</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#featured" data-target="#featured"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#projects" data-target="#projects"><span>Projects</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/files/CV_NeilDalchau.pdf"><span>CV</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link js-theme-selector" data-toggle="dropdown" aria-haspopup="true">
          <i class="fas fa-palette" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>


  









<span class="js-widget-page d-none"></span>





  
  
  
  




  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="about" class="home-section wg-about   "  >
    <div class="container">
      




  










<div class="row">
  <div class="col-12 col-lg-4">
    <div id="profile">

      
      
      <img class="avatar avatar-circle" src="/authors/admin/avatar_hu92567743ada9e228812162da2e663d3b_179754_270x270_fill_lanczos_center_3.png" alt="">
      

      <div class="portrait-title">
        <h2></h2>
        <h3>Chief Researcher Officer</h3>

        
        <h3>
          
          <span>Synteny Biotechnology</span>
          
        </h3>
        
      </div>

      <ul class="network-icon" aria-hidden="true">
        
        
        
        
        
        
        
        
          
        
        <li>
          <a href="https://scholar.google.co.uk/citations?user=bKnBEZ8AAAAJ&amp;hl=en" target="_blank" rel="noopener">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        
        
        
        
          
        
        <li>
          <a href="https://github.com/ndalchau" target="_blank" rel="noopener">
            <i class="fab fa-github big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        
        
        
        
          
        
        <li>
          <a href="https://www.linkedin.com/in/neil-dalchau-499b616/" target="_blank" rel="noopener">
            <i class="fab fa-linkedin big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        
        
        
        
          
        
        <li>
          <a href="https://twitter.com/ndalchau" target="_blank" rel="noopener">
            <i class="fab fa-twitter big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        
        
        
        
        <li>
          <a href="mailto:ndalchau@gmail.com" >
            <i class="fas fa-envelope big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-12 col-lg-8">

    
    <h1>Biography</h1>

    <p>I am the Chief Research Officer at 
<a href="http://synteny.ai" target="_blank" rel="noopener">Synteny Biotechnology</a>. Our organisation uses modern AI methodologies combined with high-throughput experimental techniques to understand the specificities of T-cells, and in doing so, enable a new generation of T-cell based therapies and diagnostics. The company was founded by Lilly Wollman and Jamie Blundell, and I joined in January 2022 to build the organisation from scratch. By working at Synteny, I am able to focus on researching the fascinating natural computation that our bodies perform to identify pathogens and dysfunctional cells.</p>
<p>Before Synteny, I was a Principal Scientist and Research Manager at 
<a href="https://www.microsoft.com/en-us/research/lab/microsoft-research-cambridge/" target="_blank" rel="noopener">Microsoft Research Cambridge</a> and project lead for 
<a href="http://aka.ms/stationb" target="_blank" rel="noopener">Station B</a>. Before that, I was a PhD student at University of Cambridge, where I worked on circadian timing in plants in the laboratory of 
<a href="https://www.plantsci.cam.ac.uk/research/groups/circadian-signal-transduction" target="_blank" rel="noopener">Alex Webb</a>.</p>
<p>During my career, I have always operated at the intersection of biological data and computational analysis. I have made biological discoveries using a wide range of computational techniques. The majority of my earlier work used ordinary differential equation (ODE) models and stochastic chemical kinetics (essentially continuous-time Markov chains). I have also developed techniques for parameter inference and parameter synthesis with dynamical models. In my later years at Microsoft Research, I became interested in probabilistic machine learning models and active learning approaches, including Bayesian optimization. At Synteny, the primary observations are of amino acid sequences, and so I have become interested in using methods that can classify functional properties of those sequences, or <em>repertoires</em> (sets) of (T-cell receptor and antigen) sequences.</p>
<p>At Synteny, I have been able to reignite my interest in Immunology, which started when I first joined Andrew Phillips&rsquo; research group at Microsoft in 2009. Together, we developed some of the first dynamical models of antigen presentation by class I molecules of the major histocompability complex (MHC), collaborating with Tim Elliott, who at the time was at the University of Southampton.</p>


    <div class="row">

      
      <div class="col-md-5">
        <h3>Interests</h3>
        <ul class="ul-interests">
          
          <li>Immunology</li>
          
          <li>Synthetic Biology</li>
          
          <li>DNA Computing</li>
          
          <li>Bayesian Inference</li>
          
          <li>Bayesian Optimization</li>
          
        </ul>
      </div>
      

      
      <div class="col-md-7">
        <h3>Education</h3>
        <ul class="ul-edu fa-ul">
          
          <li>
            <i class="fa-li fas fa-graduation-cap"></i>
            <div class="description">
              <p class="course">PhD in Plant Sciences, 2009</p>
              <p class="institution">University of Cambridge</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fas fa-graduation-cap"></i>
            <div class="description">
              <p class="course">MMath in Mathematics, 2005</p>
              <p class="institution">University of Oxford</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="industryexperience" class="home-section wg-experience   "  >
    <div class="container">
      



<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Industry Experience</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

    
    
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col ">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border exp-fill">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Chief Research Officer</h4>
            <h4 class="card-title exp-company text-muted my-0">Synteny</h4>
            <div class="text-muted exp-meta">
              Jan 2022 –
              
                Present
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Principal Research Manager</h4>
            <h4 class="card-title exp-company text-muted my-0">Microsoft Research</h4>
            <div class="text-muted exp-meta">
              Nov 2020 –
              
                Oct 2021
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            <div class="card-text">Station B sought to improve the way we engineer biological systems, bringing together computational modelling, lab automation and machine learning. My role was to lead this project, and seek opportunities for Microsoft in the biotechnology industry.</div>
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Principal Scientist</h4>
            <h4 class="card-title exp-company text-muted my-0">Microsoft Research</h4>
            <div class="text-muted exp-meta">
              Jan 2015 –
              
                Sep 2020
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Scientist</h4>
            <h4 class="card-title exp-company text-muted my-0">Microsoft Research</h4>
            <div class="text-muted exp-meta">
              Mar 2012 –
              
                Dec 2014
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Postdoctoral Researcher</h4>
            <h4 class="card-title exp-company text-muted my-0">Microsoft.Research</h4>
            <div class="text-muted exp-meta">
              Feb 2012 –
              
                Sep 2009
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            <div class="card-text">During this post-doc, I was researching the use of dynamical systems models applied to immune system processes and synthetic biology. In particular, I helped the construction of the first model of the MHC class I antigen presentation mechanism, a central player in the adaptive immune system (Dalchau et al., PLoS Computational Biology 2011). In Synthetic Biology, I constructed and analysed models of synthetic cell populations which had been modified to give rise to Turing patterns (unpublished) or periodic travelling waves (Dalchau et al., J Royal Society Interface 2012).</div>
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Research Intern</h4>
            <h4 class="card-title exp-company text-muted my-0">Microsoft Research</h4>
            <div class="text-muted exp-meta">
              Jun 2009 –
              
                Aug 2009
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            <div class="card-text">This internship was a prequel to my subsequent postdoctoral position at Microsoft Research.</div>
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col ">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Support and Applications Engineer</h4>
            <h4 class="card-title exp-company text-muted my-0">Vector Fields Ltd.</h4>
            <div class="text-muted exp-meta">
              Jul 2004 –
              
                Sep 2005
              
              
                <span class="middot-divider"></span>
                <span>Oxford, UK</span>
              
            </div>
            <div class="card-text">Vector Fields provide software for the analysis and design of electromagnetic devices. 3d and 2d models solved using Finite Element methods. I supported international clients, and provided consultancy and benchmarks for existing and prospective customers.</div>
          </div>
        </div>
      </div>
    </div>
    
    
  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="academicexperience" class="home-section wg-experience   "  >
    <div class="container">
      



<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Academic Experience</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

    
    
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col ">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border exp-fill">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Affiliate Researcher</h4>
            <h4 class="card-title exp-company text-muted my-0"><a href="http://www.kcl.ac.uk" target="_blank" rel="noopener">Kings College London</a></h4>
            <div class="text-muted exp-meta">
              Aug 2020 –
              
                Present
              
              
                <span class="middot-divider"></span>
                <span>London, UK</span>
              
            </div>
            <div class="card-text">Co-supervision of PhD student 
<a href="https://www.kcl.ac.uk/noneqsys/people-and-partners/student-profiles/gregory-szep" target="_blank" rel="noopener">Grisha Szep</a>, in collaboration with Attila Csikasz-Nagy.</div>
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Visiting Researcher</h4>
            <h4 class="card-title exp-company text-muted my-0">University of Edinburgh</h4>
            <div class="text-muted exp-meta">
              Nov 2012 –
              
                Dec 2017
              
              
                <span class="middot-divider"></span>
                <span>Edinburgh, UK</span>
              
            </div>
            <div class="card-text">Co-supervision of PhD student Yiyu Xiang, in collaboration with Baojun Wang.</div>
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Visiting Researcher</h4>
            <h4 class="card-title exp-company text-muted my-0">University College London</h4>
            <div class="text-muted exp-meta">
              Oct 2012 –
              
                Aug 2017
              
              
                <span class="middot-divider"></span>
                <span>London, UK</span>
              
            </div>
            <div class="card-text">Co-supervision of two PhD students, Laura Parshotam and Ruth Charlotte Eccleston, in collaboration with Peter Coveney. Both students successfully defended their PhD theses on time.</div>
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">Postdoctoral Researcher</h4>
            <h4 class="card-title exp-company text-muted my-0">University of Cambridge</h4>
            <div class="text-muted exp-meta">
              Nov 2008 –
              
                May 2009
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            <div class="card-text">Investigating noise suppression in gene regulatory networks, advised by Dr Glenn Vinnicombe. Experience with doubly stochastic processes, stochastic control and Markov birth-death processes.</div>
          </div>
        </div>
      </div>
    </div>
    
    <div class="row experience">
      
      <div class="col-auto text-center flex-column d-none d-sm-flex">
        <div class="row h-50">
          <div class="col border-right">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
        <div class="m-2">
          <span class="badge badge-pill border ">&nbsp;</span>
        </div>
        <div class="row h-50">
          <div class="col ">&nbsp;</div>
          <div class="col">&nbsp;</div>
        </div>
      </div>
      
      <div class="col py-2">
        <div class="card">
          <div class="card-body">
            <h4 class="card-title exp-title text-muted mt-0 mb-1">College Supervisor</h4>
            <h4 class="card-title exp-company text-muted my-0">Robinson College, University of Cambridge</h4>
            <div class="text-muted exp-meta">
              Oct 2006 –
              
                Jul 2008
              
              
                <span class="middot-divider"></span>
                <span>Cambridge, UK</span>
              
            </div>
            <div class="card-text">Taught undergraduate Natural Sciences students for the lecture course &lsquo;Quantitative Biology&rsquo; (QB) for two years.</div>
          </div>
        </div>
      </div>
    </div>
    
    
  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="accomplishments" class="home-section wg-accomplishments   "  >
    <div class="container">
      



<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Accomplishments</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

    
    
      <div class="card experience course">
        <div class="card-body"><h4 class="card-title exp-title text-muted my-0">Honorary Senior Research Associate, Department of Chemistry</h4><div class="card-subtitle my-0 article-metadata">University College London<span class="middot-divider"></span>

            2013
            
          </div>

          

          
        </div>
      </div>
    
      <div class="card experience course">
        <div class="card-body"><a href="https://www.newphytologist.org/awards/tansleymedal" target="_blank" rel="noopener"><h4 class="card-title exp-title text-muted my-0">Tansley Medal</h4></a><div class="card-subtitle my-0 article-metadata">New Phytologist<span class="middot-divider"></span>

            2011
            
          </div>

          
            <div class="card-text">The New Phytologist Tansley Medal was established in 2009 for the recognition of outstanding contributions made by scientists early in their independent career.</div>
          

          
            <a class="card-link" href="https://nph.onlinelibrary.wiley.com/doi/full/10.1111/j.1469-8137.2012.04056.x" target="_blank" rel="noopener">
              See certificate
            </a>
          
        </div>
      </div>
    
    
  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="featured" class="home-section wg-featured   "  >
    <div class="container">
      




























  


<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Featured Publications</h1>
    
  </div>
  <div class="col-12 col-lg-8">

    

    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Szép G</span>, <span >Dalchau N</span>, <span >Csikász-Nagy A</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Neural Information Processing Systems (NeurIPS)
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  
  
  
  <a href="/publication/2021-neurips-szep/">
      <img src="/publication/2021-neurips-szep/featured_hu1fd4a71654b055a71bd76ab571926180_577241_918x517_fit_lanczos_3.png" class="article-banner" alt="Parameter Inference with Bifurcation Diagrams">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2021-neurips-szep/">Parameter Inference with Bifurcation Diagrams</a>
  </h3>

  
  <a href="/publication/2021-neurips-szep/" class="summary-link">
    <div class="article-style">
      <p>Estimation of parameters in differential equation models can be achieved by applying learning algorithms to quantitative time-series data. However, sometimes it is only possible to measure qualitative changes of a system in response to a controlled condition. In dynamical systems theory, such change points are known as bifurcations and lie on a function of the controlled condition called the bifurcation diagram. In this work, we propose a gradient-based semi-supervised approach for inferring the parameters of differential equations that produce a user-specified bifurcation diagram. The cost function contains a supervised error term that is minimal when the model bifurcations match the specified targets and an unsupervised bifurcation measure which has gradients that push optimisers towards bifurcating parameter regimes. The gradients can be computed without the need to differentiate through the operations of the solver that was used to compute the diagram. We demonstrate parameter inference with minimal models which explore the space of saddle-node and pitchfork diagrams and the genetic toggle switch from synthetic biology. Furthermore, the cost landscape allows us to organise models in terms of topological and geometric equivalence.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://arxiv.org/abs/2106.04243" target="_blank" rel="noopener">
  Preprint
</a>




  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/gszep/BifurcationInference.jl" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    Github
  </a>


  </div>
  

</div>

      
    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Grant PK</span>, <span >Szép G</span>, <span >Patange O</span>, <span >Halatek J</span>, <span >Coppard V</span>, <span >Csikász-Nagy A</span>, <span >Haseloff J</span>, <span >Locke JCW</span>, <span >Dalchau N</span>, <span >Phillips A</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Communications
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  
  
  
  <a href="/publication/2020-nat-comms-grant/">
      <img src="/publication/2020-nat-comms-grant/featured_hud0db1eec88f884b51f09755e9f4a0c14_549338_918x517_fit_lanczos_3.PNG" class="article-banner" alt="Interpretation of morphogen gradients by a synthetic bistable circuit">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-nat-comms-grant/">Interpretation of morphogen gradients by a synthetic bistable circuit</a>
  </h3>

  
  <a href="/publication/2020-nat-comms-grant/" class="summary-link">
    <div class="article-style">
      <p>During development, cells gain positional information through the interpretation of dynamic morphogen gradients. A proposed mechanism for interpreting opposing morphogen gradients is mutual inhibition of downstream transcription factors, but isolating the role of this specific motif within a natural network remains a challenge. Here, we engineer a synthetic morphogen-induced mutual inhibition circuit in E. coli populations and show that mutual inhibition alone is sufficient to produce stable domains of gene expression in response to dynamic morphogen gradients, provided the spatial average of the morphogens falls within the region of bistability at the single cell level. When we add sender devices, the resulting patterning circuit produces theoretically predicted self-organised gene expression domains in response to a single gradient. We develop computational models of our synthetic circuits parameterised to timecourse fluorescence data, providing both a theoretical and experimental framework for engineering morphogen-induced spatial patterning in cell populations.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/https://doi.org/10.1038/s41467-020-19098-w" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/gszep/double-exclusive-reporter" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    Github
  </a>


  </div>
  

</div>

      
    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Roeder G</span>, <span >Grant PK</span>, <span >Phillips A</span>, <span >Dalchau N</span>, <span >Meeds E</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      International Conference on Machine Learning (ICML)
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  
  
  
  <a href="/publication/2019-icml-roeder/">
      <img src="/publication/2019-icml-roeder/featured_hud89140a1e0767f55e7cbbd1900797e8c_62361_918x517_fit_lanczos_3.png" class="article-banner" alt="Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2019-icml-roeder/">Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems</a>
  </h3>

  
  <a href="/publication/2019-icml-roeder/" class="summary-link">
    <div class="article-style">
      <p>We introduce a flexible, scalable Bayesian inference framework for nonlinear dynamical systems characterised by distinct and hierarchical variability at the individual, group, and population levels. Our model class is a generalisation of nonlinear mixed-effects (NLME) dynamical systems, the statistical workhorse for many experimental sciences. We cast parameter inference as stochastic optimisation of an end-to-end differentiable, block-conditional variational autoencoder. We specify the dynamics of the data-generating process as an ordinary differential equation (ODE) such that both the ODE and its solver are fully differentiable. This model class is highly flexible: the ODE right-hand sides can be a mixture of user-prescribed or &quot;white-box&quot; sub-components and neural network or &quot;black-box&quot; sub-components. Using stochastic optimisation, our amortised inference algorithm could seamlessly scale up to massive data collection pipelines (common in labs with robotic automation). Finally, our framework supports interpretability with respect to the underlying dynamics, as well as predictive generalization to unseen combinations of group components (also called &quot;zero-shot&quot; learning). We empirically validate our method by predicting the dynamic behaviour of bacteria that were genetically engineered to function as biosensors.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://arxiv.org/abs/1905.12090" target="_blank" rel="noopener">
  Preprint
</a>




  









  
    
  










  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/Biological-Computation/vi-hds" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    Github
  </a>


  </div>
  

</div>

      
    

  

  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="publications" class="home-section wg-pages   "  >
    <div class="container">
      








  
























  





  


<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Recent Publications</h1>
    
  </div>
  <div class="col-12 col-lg-8">

    

    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Boulanger DS</span>, <span >Douglas LR</span>, <span >Duriez PJ</span>, <span >Kang Y</span>, <span >Dalchau N</span>, <span >James E</span>, <span >Elliott T</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Immunology
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2023-tapasin-boulanger/">Tapasin-mediated editing of the MHC I immunopeptidome is epitope specific and dependent on peptide off-rate, abundance, and level of tapasin expression</a>
  </h3>

  
  <a href="/publication/2023-tapasin-boulanger/" class="summary-link">
    <div class="article-style">
      <p>Tapasin, a component of the major histocompatibility complex (MHC) I peptide loading complex, edits the repertoire of peptides that is presented at the cell surface by MHC I and thereby plays a key role in shaping the hierarchy of CD8+ T-cell responses to tumors and pathogens. We have developed a system that allows us to tune the level of tapasin expression and independently regulate the expression of competing peptides of different off-rates. By quantifying the relative surface expression of peptides presented by MHC I molecules, we show that peptide editing by tapasin can be measured in terms of “tapasin bonus,” which is dependent on both peptide kinetic stability (off-rate) and peptide abundance (peptide supply). Each peptide has therefore an individual tapasin bonus fingerprint. We also show that there is an optimal level of tapasin expression for each peptide in the immunopeptidome, dependent on its off-rate and abundance. This is important, as the level of tapasin expression can vary widely during different stages of the immune response against pathogens or cancer and is often the target for immune escape.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/https://doi.org/10.3389/fimmu.2022.956603" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Fil J</span>, <span >Dalchau N</span>, <span >Chu D</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      ACS Synthetic Biology
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  
  
  
  <a href="/publication/2022-acs-sb-fil/">
      <img src="/publication/2022-acs-sb-fil/featured_hu6f823811223efde61c1cbb347ddc308d_73215_918x517_fit_q90_lanczos.jpeg" class="article-banner" alt="Programming Molecular Systems To Emulate a Learning Spiking Neuron">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2022-acs-sb-fil/">Programming Molecular Systems To Emulate a Learning Spiking Neuron</a>
  </h3>

  
  <a href="/publication/2022-acs-sb-fil/" class="summary-link">
    <div class="article-style">
      <p>Hebbian theory seeks to explain how the neurons in the brain adapt to stimuli to enable learning. An interesting feature of Hebbian learning is that it is an unsupervised method and, as such, does not require feedback, making it suitable in contexts where systems have to learn autonomously. This paper explores how molecular systems can be designed to show such protointelligent behaviors and proposes the first chemical reaction network (CRN) that can exhibit autonomous Hebbian learning across arbitrarily many input channels. The system emulates a spiking neuron, and we demonstrate that it can learn statistical biases of incoming inputs. The basic CRN is a minimal, thermodynamically plausible set of microreversible chemical equations that can be analyzed with respect to their energy requirements. However, to explore how such chemical systems might be engineered de novo, we also propose an extended version based on enzyme-driven compartmentalized reactions. Finally, we show how a purely DNA system, built upon the paradigm of DNA strand displacement, can realize neuronal dynamics. Our analysis provides a compelling blueprint for exploring autonomous learning in biological settings, bringing us closer to realizing real synthetic biological intelligence.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/https://doi.org/10.1021/acssynbio.1c00625" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Arce A</span>, <span >Guzman-Chavez F</span>, <span >Gandini C</span>, <span >Puig J</span>, <span >Matute T</span>, <span >Haseloff J</span>, <span >Dalchau N</span>, <span >Molloy J</span>, <span >Pardee K</span>, <span >Federici F</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Bioengineering and Biotechnology
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2021-frontiers-arce/">Decentralizing Cell-Free RNA Sensing With the Use of Low-Cost Cell Extracts</a>
  </h3>

  
  <a href="/publication/2021-frontiers-arce/" class="summary-link">
    <div class="article-style">
      <p>Cell-free gene expression systems have emerged as a promising platform for field-deployed biosensing and diagnostics. When combined with programmable toehold switch-based RNA sensors, these systems can be used to detect arbitrary RNAs and freeze-dried for room temperature transport to the point-of-need. These sensors, however, have been mainly implemented using reconstituted PURE cell-free protein expression systems that are difficult to source in the Global South due to their high commercial cost and cold-chain shipping requirements. Based on preliminary demonstrations of toehold sensors working on lysates, we describe the fast prototyping of RNA toehold switch-based sensors that can be produced locally and reduce the cost of sensors by two orders of magnitude. We demonstrate that these in-house cell lysates provide sensor performance comparable to commercial PURE cell-free systems. We further optimize these lysates with a CRISPRi strategy to enhance the stability of linear DNAs by knocking-down genes responsible for linear DNA degradation. This enables the direct use of PCR products for fast screening of new designs. As a proof-of-concept, we develop novel toehold sensors for the plant pathogen Potato Virus Y (PVY), which dramatically reduces the yield of this important staple crop. The local implementation of low-cost cell-free toehold sensors could enable biosensing capacity at the regional level and lead to more decentralized models for global surveillance of infectious disease.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/https://doi.org/10.3389/fbioe.2021.727584" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Zhang J</span>, <span >Yordanov B</span>, <span >Gaunt A</span>, <span >Wang M</span>, <span >Dai P</span>, <span >Chen Y-J</span>, <span >Zhang K</span>, <span >Fang J</span>, <span >Dalchau N</span>, <span >Li J</span>, <span >Phillips A</span>, <span >Zhang D</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Communications
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2021-nat-comms-zhang/">A Deep Learning Model for Predicting NGS Sequencing Depth from DNA Sequence</a>
  </h3>

  
  <a href="/publication/2021-nat-comms-zhang/" class="summary-link">
    <div class="article-style">
      <p>Targeted high-throughput DNA sequencing is a primary approach for genomics and molecular diagnostics, and more recently as a readout for DNA information storage. Oligonucleotide probes used to enrich gene loci of interest have different hybridization kinetics, resulting in non-uniform coverage that increases sequencing costs and decreases sequencing sensitivities. Here, we present a deep learning model (DLM) for predicting Next-Generation Sequencing (NGS) depth from DNA probe sequences. Our DLM includes a bidirectional recurrent neural network that takes as input both DNA nucleotide identities as well as the calculated probability of the nucleotide being unpaired. We apply our DLM to three different NGS panels: a 39,145-plex panel for human single nucleotide polymorphisms (SNP), a 2000-plex panel for human long non-coding RNA (lncRNA), and a 7373-plex panel targeting non-human sequences for DNA information storage. In cross-validation, our DLM predicts sequencing depth to within a factor of 3 with 93% accuracy for the SNP panel, and 99% accuracy for the non-human panel. In independent testing, the DLM predicts the lncRNA panel with 89% accuracy when trained on the SNP panel. The same model is also effective at predicting the measured single-plex kinetic rate constants of DNA hybridization and strand displacement.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/https://doi.org/10.1038/s41467-021-24497-8" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Szép G</span>, <span >Dalchau N</span>, <span >Csikász-Nagy A</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Neural Information Processing Systems (NeurIPS)
    
  </span>
  

  

  
  
  

  

</div>

  

  
  
  
  
  
  <a href="/publication/2021-neurips-szep/">
      <img src="/publication/2021-neurips-szep/featured_hu1fd4a71654b055a71bd76ab571926180_577241_918x517_fit_lanczos_3.png" class="article-banner" alt="Parameter Inference with Bifurcation Diagrams">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2021-neurips-szep/">Parameter Inference with Bifurcation Diagrams</a>
  </h3>

  
  <a href="/publication/2021-neurips-szep/" class="summary-link">
    <div class="article-style">
      <p>Estimation of parameters in differential equation models can be achieved by applying learning algorithms to quantitative time-series data. However, sometimes it is only possible to measure qualitative changes of a system in response to a controlled condition. In dynamical systems theory, such change points are known as bifurcations and lie on a function of the controlled condition called the bifurcation diagram. In this work, we propose a gradient-based semi-supervised approach for inferring the parameters of differential equations that produce a user-specified bifurcation diagram. The cost function contains a supervised error term that is minimal when the model bifurcations match the specified targets and an unsupervised bifurcation measure which has gradients that push optimisers towards bifurcating parameter regimes. The gradients can be computed without the need to differentiate through the operations of the solver that was used to compute the diagram. We demonstrate parameter inference with minimal models which explore the space of saddle-node and pitchfork diagrams and the genetic toggle switch from synthetic biology. Furthermore, the cost landscape allows us to organise models in terms of topological and geometric equivalence.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://arxiv.org/abs/2106.04243" target="_blank" rel="noopener">
  Preprint
</a>




  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/gszep/BifurcationInference.jl" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    Github
  </a>


  </div>
  

</div>

      
    

    
    <div class="see-all">
      <a href="/publication/">
        See all publications
        <i class="fas fa-angle-right"></i>
      </a>
    </div>
    

  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="projects" class="home-section wg-portfolio   "  >
    <div class="container">
      










<div class="row">
  <div class="col-12 col-lg-4 section-heading">

    <h1>Projects</h1>
    

  </div>
  <div class="col-12 col-lg-8">



    

    

    <div class=" row js-layout-row project-showcase mt-5">
      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item ">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0"><a href="/project/vihds/" >Variational inference for ODE models</a></h3>

      
      <div class="article-style">
        Variational autoencoders can be used to infer hierarchical (global, group-level and individual-level) parameters of  dynamical systems models (ordinary differential equations), and leverages gradient-based optimization
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://github.com/Biological-Computation/vi-hds" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    vi-hds
  </a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://patents.justia.com/patent/11030275" target="_blank" rel="noopener">
    <i class="fas fa-book mr-1"></i>
    Patent
  </a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://www.microsoft.com/en-us/research/blog/efficient-inference-for-dynamical-models-using-variational-autoencoders/" target="_blank" rel="noopener">
    <i class="fas fa-blog mr-1"></i>
    Microsoft Blog
  </a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
      
      <a href="/project/vihds/" >
        <img src="/project/vihds/featured_huab75be2ab86fd8b10cd8195595a98d37_58305_540x0_resize_lanczos_3.png" alt="">
      </a>
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item ">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0"><a href="/project/dna/" >DNA Computing</a></h3>

      
      <div class="article-style">
        Programming DNA molecules offers the potential for applications from bio-sensing to intelligent nanomedicine
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://github.com/Biological-Computation/crn-engine" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    crn-engine
  </a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://ph1ll1ps.github.io/visualdsd/index.html" target="_blank" rel="noopener">
    <i class="fas fa-globe mr-1"></i>
    Visual DSD
  </a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
      
      <a href="/project/dna/" >
        <img src="/project/dna/featured_hu06f18105299878cbf4c302ae9f9d6020_119731_540x0_resize_q90_lanczos.jpg" alt="">
      </a>
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item ">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0"><a href="/project/crn/" >Chemical Reaction Networks</a></h3>

      
      <div class="article-style">
        Tools for programming chemical reaction networks (CRN), DNA strand-displacement circuits (DSD) and genetically engineered circuits (GEC)
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://github.com/Biological-Computation/crn-engine" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    crn-engine
  </a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item ">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0"><a href="/project/synbio/" >Synthetic Biology</a></h3>

      
      <div class="article-style">
        Synthetic biology aims to use engineering principles to augment natural cells with the ability to carry out well-defined functions that confer industrial or societal benefits
      </div>
      

      <div class="btn-links">
        








  


















  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://github.com/Biological-Computation/crn-engine" target="_blank" rel="noopener">
    <i class="fab fa-github mr-1"></i>
    crn-engine
  </a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1" href="https://classicgec.azurewebsites.net/" target="_blank" rel="noopener">
    <i class="fas fa-globe mr-1"></i>
    Visual GEC
  </a>


      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
      
      <a href="/project/synbio/" >
        <img src="/project/synbio/featured_hu0a9b469bdb608b076f104b3eeb335b1d_135677_540x0_resize_q90_lanczos.jpg" alt="">
      </a>
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item js-id-test">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0"><a href="/project/immunology/" >Immunology</a></h3>

      
      <div class="article-style">
        Computational modelling of immune system processes
      </div>
      

      <div class="btn-links">
        








  



















      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item ">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0">Plant Biology</h3>

      
      <div class="article-style">
        My research in plant biology includes circadian clocks and how the decision to flower depends on carbon-nitrogen availability
      </div>
      

      <div class="btn-links">
        








  



















      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
    </div>
  </div>
</div>

        

      
    </div>

  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="people" class="home-section wg-people   "  >
    <div class="container">
      







<div class="row justify-content-center people-widget">
  
  <div class="col-md-12 section-heading">
    <h1>People</h1>
    <p>People I&rsquo;ve Worked Closely With</p>
  </div>
  

  

  
  

  
  <div class="col-md-12">
    <h2 class="mb-4">Microsoft Research</h2>
  </div>
  

  
  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      
      <a href="/authors/aphillip/"><img class="avatar avatar-circle" src="/authors/aphillip/avatar_hu47d34ebc8cea172c4e8698e34d2ef49e_12772_270x270_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/aphillip/"></a></h2>
      <h2>Andrew Phillips</h2>
      <h3>Innovator and leader in molecular and genetic programming</h3>
      <ul class="network-icon" aria-hidden="true">
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://scholar.google.co.uk/citations?user=-0dWVY4AAAAJ&amp;hl=en" target="_blank" rel="noopener">
        <i class="ai ai-google-scholar"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://github.com/ph1ll1ps" target="_blank" rel="noopener">
        <i class="fab fa-github"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://www.linkedin.com/in/ph1ll1ps" target="_blank" rel="noopener">
        <i class="fab fa-linkedin"></i>
      </a>
    </li>
  
</ul>

      
    </div>
  </div>
  
  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      
      <a href="/authors/cgravill/"><img class="avatar avatar-circle" src="/authors/cgravill/avatar_hu7ea5ba7ead27766d2ec584de96861d69_32262_270x270_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/cgravill/"></a></h2>
      <h2>Colin Gravill</h2>
      <h3>Software Developer</h3>
      <ul class="network-icon" aria-hidden="true">
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://github.com/cgravill" target="_blank" rel="noopener">
        <i class="fab fa-github"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://www.linkedin.com/in/colingravill/" target="_blank" rel="noopener">
        <i class="fab fa-linkedin"></i>
      </a>
    </li>
  
</ul>

      
    </div>
  </div>
  
  
</div>

    </div>
  </section>



      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js" integrity="sha256-eOgo0OtLL4cdq7RdwRUiGKLX9XsIJ7nGhWEKbohmVAQ=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/r.min.js"></script>
        
      

    

    
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = true;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks",
        'slides' : "Slides"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    
    <script>
      if (window.netlifyIdentity) {
        window.netlifyIdentity.on("init", user => {
          if (!user) {
            window.netlifyIdentity.on("login", () => {
              document.location.href = "/admin/";
            });
          }
        });
      }
    </script>
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.0d53a7f39b7d6770128a54b39dad2f22.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    
  </p>

  
  






  <p class="powered-by">
    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
